### ISSUE TITLE
H30: Reported error is not an error.

### ISSUE THREAD

**Issue by RaphDeRooij** [squizlabs/HTML_CodeSniffer#52]
When running HTML_CodeSniffer on http://www.w3.org/WAI/, the app reports one error:
"Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link."

Issue: the IMG element is NOT the only content of the link.
